### PR TITLE
feature: ZENKO-3134-update-vault-image-v8.2.0-alpha.1

### PR DESCRIPTION
**Pull request by anurag4DSB**
**What does this PR do, and why do we need it?**
Update Vault image with v8.2.0-alpha.1

**Special notes for your reviewers**:
Image pull command: `docker pull registry.scality.com/vault/vault:8.2.0-alpha.1`

For instructions on docker login: https://github.com/scality/Vault/blob/development/8.2/docs/RELEASE.md#how-to-pull-docker-images

Respective Operator PR merged: https://github.com/scality/zenko-operator/pull/128/files

### CODE REVIEW

**Comment by anurag4DSB** [comment]
ping

**Comment by tcarmet** [comment]
I believe we also need to change the registry url in the file `solution/deps.txt`

**Comment by miniscruff** [comment]
ZENKO-3155 is going to update all the images, plus I think we still need to update zenko-operator to handle the new vault before putting it in zenko, which is ZENKO-3146 that I will be working on starting tomorrow.

**Comment by anurag4DSB** [comment]
> [ZENKO-3155](https://scality.atlassian.net/browse/ZENKO-3155)

@terrydervaux as this will be done in Zenko-3155
Should I close my ticket?

**Comment by anurag4DSB** [comment]
> I believe we also need to change the registry url in the file `solution/deps.txt`

@tcarmet changed the registry URL here in the squash commit :)

**Comment by anurag4DSB** [comment]
> [ZENKO-3155](https://scality.atlassian.net/browse/ZENKO-3155) is going to update all the images, plus I think we still need to update zenko-operator to handle the new vault before putting it in zenko, which is [ZENKO-3146](https://scality.atlassian.net/browse/ZENKO-3146) that I will be working on starting tomorrow.

@miniscruff This should not block me from updating vault dependency. If the zenko CI failed because of the operator or some other reason, yes the PR should not be merge until that thing is fixed. So I do not see a reason why this PR should be blocked or put on hold till we update the operator.

If CI is green we should be able to merge our dep update. So that we can move on to the next thing.

That said I know the CI is not in place properly yet. For now Zenko2.x will not work(already doesn't) and the CI is not reliable. But we will get there :) 

Per the process discussed in the various meetings, I should be able to update my dependency if it is ready. What do you think? :)

**Comment by miniscruff** [comment]
> If CI is green we should be able to merge our dep update. So that we can move on to the next thing.

The build will not fail because the e2e test does not use either of the files you updated. That config is placed inside a shell script so your change is not being tested at all. ( Updating this file is also related to the issue I posted and will be merged into the single source later on ).

> Per the process discussed in the various meetings, I should be able to update my dependency if it is ready. What do you think? :)

Typically yes, but, and correct me if I am wrong here, your latest change requires a new configuration. Meaning, if we do not add this new config vault will not work properly. That was my concern.

**Comment by anurag4DSB** [comment]
forced pushed review commits and new vault version with fixes.

@miniscruff I agree with you, in a real world scenario(near future) when zenko-CI is working, I won't update vault Image till dependencies like Operator is also updated.

PS: Second thoughts...

**Comment by anurag4DSB** [comment]
The pending PR has been merged: https://github.com/scality/zenko-operator/pull/128/files

File: solution/zenkoversion.yaml [hunk 1]
```diff
@@ -50,8 +50,8 @@ spec:
       image: kafka
       tag: 2.12-2.3.0
     vault:
-      image: vault
-      tag: zenko-2.0.0
+      image: ${SOLUTION_REGISTRY}/vault
```
Comment (miniscruff) [inline solution/zenkoversion.yaml:53]:
```suggestion
      image: vault
```

Comment (anurag4DSB) [reply solution/zenkoversion.yaml:53]: I was following what zenko-ui has on line # 11 of the same file.
and the same is mentioned in the summary of this ticket: https://scality.atlassian.net/browse/ZENKO-3134

If I specify just vault, won't it look for it in dockerhub and not solution registry?

CC 
@tcarmet for confirmation as he reported the ticket.
@nicolas2bert for awareness

**Comment by anurag4DSB** [comment]
Update: closing this PR as ZENKO-3163 will handle image update of vault as well (per discussion with @miniscruff)

**Pull request closed without merging**